hazelcast/src/test/java/com/hazelcast/instance/HazelcastInstanceFactoryTest.java

```diff
@@ -177,24 +190,6 @@ public Object answer(InvocationOnMock invocation) throws Throwable {
         }
     }
 
-    @Test(expected = ExpectedRuntimeException.class)
-    public void test_NewInstance_failedAfterStartAndBeforeShutdown() throws Exception {
-        NodeContext context = new TestNodeContext() {
-            @Override
-            public NodeExtension createNodeExtension(Node node) {
-                NodeExtension nodeExtension = super.createNodeExtension(node);
-                doThrow(new ExpectedRuntimeException()).when(nodeExtension).afterStart();
```

Comment:
`NodeExtension.afterStart()` is already tested like this in `test_NewInstance_failed_afterNodeStart()`